vm/core/context.go

```diff
@@ -363,30 +366,29 @@ func (c *Context) Has(address types.Address) (bool, error) {
 	return true, nil
 }
 
-func (c *Context) Get(address types.Address) (Account, error) {
-	addr, err := c.load(address)
-	return *addr, err
+func (c *Context) Get(address types.Address) (*Account, error) {
+	return c.load(address)
 }
 
 func (c *Context) load(address types.Address) (*Account, error) {
-	if c.changed == nil {
-		c.changed = map[Address]*Account{}
-	}
 	account, exist := c.changed[address]
-	if !exist {
-		loaded, err := c.Loader.Get(address)
-		if err != nil {
-			return nil, fmt.Errorf("%w: error loading account: %w", ErrInternal, err)
-		}
-		account = &loaded
+	if exist {
+		return account, nil
 	}
-	return account, nil
+	loaded, err := c.Loader.Get(address)
+	if err != nil {
+		return nil, fmt.Errorf("%w: error loading account: %w", ErrInternal, err)
+	}
+	return &loaded, nil
 }
 
 func (c *Context) change(account *Account) {
 	_, exist := c.changed[account.Address]
 	if !exist {
 		c.touched = append(c.touched, account.Address)
 	}
+	if c.changed == nil {
```

Comment:
Is that needed? There should be no `nil` during run time.
It would be set to an empty map by `New`, or to shallow copy of some map (non nil) on `.clone()`.
The only way to have there `nil` would be by instantiating the struct without use of `New(...)`.

Reply:
It's not needed - removed.